index.d.ts

```diff
@@ -0,0 +1,21 @@
+export { ALLOWED_META } from './allowed-meta'
+export { Server, LoguxServerOptions } from './server'
+export { ServerClient } from './server-client'
+export {
+  BaseServer,
+  LoguxActionCallbacks,
+  LoguxActionFinally,
+  LoguxAuthenticator,
```

Comment:
Do we need to export all used types?

Reply:
This is optional. But there are many cases where it is convenient for developers to import types in parts so that they can be used in their code. 

For example:
```ts
import { LoguxChannelAuthorizer } from '@logux/server';

const myAcessChecker:LoguxChannelAuthorizer = (ctx, action, meta) => {
  // ...
};

server.channel('foo', {
  access: myAcessChecker,
});

server.channel('bar', {
  access: myAcessChecker,
});
```